x/username/handlers.go

```diff
@@ -0,0 +1,161 @@
+package username
+
+import (
+	"github.com/iov-one/weave"
+	"github.com/iov-one/weave/errors"
+	"github.com/iov-one/weave/migration"
+	"github.com/iov-one/weave/orm"
+	"github.com/iov-one/weave/x"
+)
+
+const (
+	registerTokenCost     = 0
+	changeTokenOwnerCost  = 0
+	changeTokenTargetCost = 0
+)
+
+func RegisterRoutes(r weave.Registry, auth x.Authenticator) {
+	r = migration.SchemaMigratingRegistry("username", r)
+
+	b := NewTokenBucket()
+	r.Handle(RegisterTokenMsg{}.Path(), &registerTokenHandler{auth: auth, bucket: b})
```

Comment:
I like use of. Path rather than reusing a constant. Cleaner and less error prone